src/app/shared/utils/fields.utils.ts

```diff
@@ -122,4 +122,13 @@ export class FieldsUtils {
     return Object.assign({}, obj);
   }
 
+  showGrayBar(caseField: CaseField, element: any) {
```

Comment:
Instead of `any`, I believe the element should be typed as `HTMLElement`.

Reply:
Done.